### ISSUE TITLE
[feature request] Multiple var declare indentation

### ISSUE THREAD

**Issue by nwhite89** [jscs-dev/node-jscs#454]
So currently I have the requireMultipleVarDecl on personally I'd like to be able to ensure that they are correctly indented also.

Example:
For it to test that this is correct
```js
var x = 'something',
    y = 'another',
    u = null;
```

This then to throw errors
```js
var x = 'something',
y = 'another',
u = null;
```

Based on the indentation set within the config also

**Comment by markelog** [comment]
No longer needed for `jquery` preset. 

Is anyone still interested in this?

**Comment by hzoo** [comment]
Cleaning up some older/inactive issues: going to close and mark as `orphaned` so we can reopen again at a later time if anyone has PR

**Comment by markelog** [comment]
I'm gonna take this one

**Comment by hzoo** [comment]
We released [3.0](https://github.com/jscs-dev/node-jscs/releases/tag/v3.0.0) recently! We also wrote a [post](https://medium.com/@markelog/jscs-end-of-the-line-bc9bf0b3fdb2#.oot3wvgpu) about how 3.0 will be our last major release (since we are deprecating the project) and that we are [merging with ESLint](http://eslint.org/blog/2016/04/welcoming-jscs-to-eslint)!

Thus we won't be working on any new features moving forward which includes new rules/options. We will continue to fix bugs and work on CST/better autofixing for current rules in the upcoming months both to help JSCS users and to gain experience that we can hopefully use in ESLint. But otherwise we both teams will be working on getting [ESLint feature parity with JSCS](https://github.com/eslint/eslint/milestones/JSCS%20Compatibility) and just a nice transition for our users.

I'm putting an `orphaned` label to note this issue isn't planned to be worked on, but might be a good contribution as a custom rule (not in JSCS core itself).

I would encourage anyone who wants the rule to try writing it themselves: I would either write a custom rule or submit an issue to ESLint and if accepted a PR. Otherwise a custom rule that won't be in core repo itself.

JSCS rules are also very similar to ESLint rules: here are some resources to get started: 

- [Project-specific Lint Rules with ESLint](https://gist.github.com/jareware/7179093)
- [Creating Custom Rules for ESLint](https://insideops.wordpress.com/2015/12/08/creating-custom-rules-for-eslint)
- [Maintaining JavaScript Code Quality with ESLint ](https://www.paypal-engineering.com/2014/12/12/maintaining-javascript-code-quality-with-eslint/)
